src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php

```diff
@@ -98,7 +98,7 @@ protected function getCapabilitiesNode()
                 ->scalarNode('platform')->defaultValue('ANY')->end()
                 ->scalarNode('browserVersion')->defaultValue('9')->end()
                 ->scalarNode('browser')->defaultValue('firefox')->end()
-                ->scalarNode('ignoreZoomSetting')->defaultValue('false')->end()
+                ->booleanNode('ignoreZoomSetting')->defaultFalse()->end()
```

Comment:
i.e. Having the following yaml:

```yml
default:
  suites:
    default:
      contexts:
        - Behat\MinkExtension\Context\MinkContext
  extensions:
    Behat\MinkExtension:
      javascript_session: default
        sessions:
          default:
            selenium2:
              browser: "internet explorer"
              capabilities: { ignoreZoomSetting: true } # or true or false doesn't matter here
```

Also fixes the issue. But not the source of the problem.